src/components/TasksMapView.js

```diff
@@ -264,6 +268,14 @@ class TasksMapView extends Component {
             onClusterPress={ this.onClusterPress }
             { ...otherProps }>
             { this.props.children }
+            {
+              this.props.isPolylineOn &&
+              (<Polyline 
+                coordinates={this.props.tasks.map(task => ( task.address.geo ))}
+                strokeWidth={3}
+                strokeColor={blueColor}
+            ></Polyline>)
+            }
```

Comment:
I suggest using a ternary operator as it's less error prone: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

```
this.props.isPolylineOn ?
(<Polyline 
  coordinates={this.props.tasks.map(task => ( task.address.geo ))}
  strokeWidth={3}
  strokeColor={blueColor}
/>) : null
```

Reply:
OK I will edit